### PR TITLE
Improving Actuator Visuals

### PR DESCRIPTION
**Pull request by rschloen**
Various changes to visuals mainly for actuator arrays including:

- orientation
- background
- body location options
- different scales to overlay on option buttons
- different number of hexagons(stichables)

### CODE REVIEW

File: src/routes/Manual.svelte [hunk 1]
```diff
@@ -77,7 +77,7 @@ $: single_pulse_duration = 500; //low priority
 
 
 $: $single_pulse_block = [0,0,0,0];
-
+//set freq to 1kHz (start with 200Hz)
```
Comment (trueb2) [inline src/routes/Manual.svelte:84]: seems irrelevant

**Review by trueb2 — COMMENTED** [review]
Are you fine with changing the target base of this PR to be user/rschloen/touch-handling? There is too much overlap to merge independently.

File: src/Utils/Hexagons.svelte [hunk 2]
```diff
@@ -344,83 +386,136 @@ const sleep = (milliseconds) => {
...
+            >
+            {#each drawableHexagons as hexagon}
+                <g key={`g-${hexagon.id}`}>
+                    <polygon
+                        id={`hex-${hexagon.id}`}
+                        key={`hex-${hexagon.x}-${hexagon.y}`}
+                        class="activatable"
+                        fill="none"
+                        stroke={hexagon.color}
+                        stroke-width={strokeWidth}
```
Comment (trueb2) [inline src/Utils/Hexagons.svelte:426]: Does this property actual get used? We had to change to camel case for stroke join.

File: src/Utils/Hexagons.svelte [hunk 3]
```diff
@@ -344,83 +386,136 @@ const sleep = (milliseconds) => {
...
+                            <text id={`text-${hexagon.id}`} x={hexagon.x - 6} y={hexagon.y}>{hexagon.id}</text>
+                        {/if}
+                    </g>
+            {/each}
+        </svg>
+        {#if test_ui}
+            {#each hexagonsLayout as actuator}
+                <button on:click={buildCommandBlocks(actuator.id)}> Button {actuator.id}</button>
+            {/each}
+            <p>Test: {$block0_31}, {$block32_63}, {$block64_95}, {$block96_127}</p>
```
Comment (trueb2) [inline src/Utils/Hexagons.svelte:440]: Do the tests still work?

**Review by trueb2 — APPROVED** [review]

File: src/Utils/Hexagons.svelte (hunk 2 above)
Comment (rschloen) [reply src/Utils/Hexagons.svelte:426]: The camel case and dash for the properties does seem inconsistent, but stroke-width was what actually worked.

File: src/Utils/Hexagons.svelte (hunk 3 above)
Comment (rschloen) [reply src/Utils/Hexagons.svelte:440]: No. I need to go back and fix them. I was waiting till I fixed multitouch in case that changed how the commands were built.
